compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -272,6 +273,8 @@ int entry(int argc, char **argv)
     options->enable(Algorithms::FoldShape);
   if (arser.get<bool>("--fold_sparse_to_dense"))
     options->enable(Algorithms::FoldSparseToDense);
+  if (arser.get<bool>("--fold_squeeze"))
```

Comment:
This new pass is introduced to optimize the below pattern after quantization.

![image](https://github.com/Samsung/ONE/assets/5449554/277375c5-627f-42d4-85a8-9989e050776d)

`Constant (fp32) -> ONNXQuantizeLinear -> ONNXDequantizeLinear -> Squeeze -> Squeeze -> Gather -> ...`

After `QuantizeOnnxQDQPass`,

`Constant (int16) -> Squeeze -> Squeeze -> Gather -> ...`

After `FoldSqueezePass`

`Constant (int16) -> Gather -> ...`